shared/CloudBackupAndRestoreUtils.ts

```diff
@@ -168,9 +211,30 @@ class Cloud {
     try {
       if (isIOS()) {
         const isSignedIn = await CloudStorage.isCloudAvailable();
-        return {
-          isSignedIn,
-        };
+
+        const userIdentifier = await RNSecureStorage.get('userIdentifier');
+        const userToken = JSON.parse(userIdentifier + '');
+        const user = userToken.user;
+        const email = userToken.email;
+
+        const credentialState = await appleAuth.getCredentialStateForUser(user);
+        let profile = {email: email, picture: undefined};
```

Comment:
Can we prefer const over let if we are not modifying this profile variable?
Also can we rename it to profileInfo since Android part has named it as profileInfo, to maintain consistency in naming?

Reply:
renamed